app/vendors/shells/sphinx_conf.php

```diff
@@ -240,7 +240,7 @@ function($v) { return $v != 'A..Z->a..z' && $v != 'a..z'; }
          * to search if they are ignored. Since all the Russian diacritics
          * are not single characters but combining characters (e.g. и + ´ = и́)
          * we simply ignore the *´* combining char (U+301) so that characters
-         * are considered not having a diacritic. */
+         * are considered as not having a diacritic. We also ignore soft hyphen. */
```

Comment:
Those are Hebrew/Yiddish combining characters (vowels). No one should ever be using them with Russian characters, or searching for them in Russian, so even if we don't explicitly ignore them in the Russian search, we shouldn't run into problems. However, if you can tell me a way to do so without code duplication, I could make that change in a later pull request.